apps/condo/domains/organization/utils/accessSchema.js

```diff
@@ -28,6 +28,26 @@ async function checkOrganizationPermission (userId, organizationId, permission)
     return employeeRole[permission] || false
 }
 
+async function checkOrganizationEmployeePermission (userId, organizationId) {
```

Comment:
The name will be confusing for developers. You check here that the user belongs to the organization even without roles
and no permissions check.

Reply:
If you just look at the name, then I would ask how it differs from `checkOrganizationPermission`.
It's confusing me too

Reply:
@pahaz, @dkoviazin I renamed method and remove deletedAt field from access query